.changeset/stale-ducks-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Fixes an issue that causes static entrypoints build to fail when the entrypoint path contains `.astro` times.
```

Comment:
What does `"contains `.astro` times."` mean?

Reply:
Imagine you have such a path `./.astro/index.astro`. The function was replacing the first `.astro` with `@_@astro`, not the extension. I agree that this may worded better tho, suggestions welcome!